eos/utils/concrete_observable.hh

```diff
@@ -127,15 +127,19 @@ namespace eos
 
             std::array<const std::string, sizeof...(Args_)> _kinematics_names_array;
 
+            Options _forced_options;
```

Comment:
See observable-impl.hh: Do you want to split off this file as a separate commit -> is independent of B->P l v ?!

Reply:
Keep it